### PR TITLE
Creates two html files - Semantic - in html folder

### PR DESCRIPTION
**Pull request by juancho11gm**
HTML Structure with Semantic and accessibility.

### CODE REVIEW

File: git/juansebastianform.html [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+          <label for="email">Email:</label>
+          <input type="email" id="email" name="email"><br>
+          <label for="dateob">Date of birth:</label>
+          <input type="date" id="dateob" name="dateob"><br>
+          <label for="gender">Gender:</label>
+          <select>
+            <option value="male" selected>Male</option>
+            <option value="female">Female</option>
+            <option value="both">Both</option>
+          </select>
```
Comment (fabianmarinog) [inline git/juansebastianform.html]: These fields could be wrapped in another `<fieldset>`

File: git/caav98.html [hunk 2]
```diff
@@ -0,0 +1,10 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title>Cesar Avila</title>
+  </head>
+  <body>
+    <h1>Cesar Ávila's git changes</h1>
+  </body>
+</html>
```
Comment (fabianmarinog) [inline git/caav98.html]: why you're including these files? Every developer should include its own files.

File: git/diego-mateus.html [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+		
+		
+		</aside>
+		
+		<footer>
+		
+		
+		</footer>
+	</body>
+</html>
```
Comment (fabianmarinog) [inline git/diego-mateus.html]: check my comment above.

File: git/caav98.html (hunk 2 above)
Comment (juancho11gm) [reply git/caav98.html]: You're right!

File: git/diego-mateus.html (hunk 3 above)
Comment (juancho11gm) [reply git/diego-mateus.html]: Sorry, I've changed it.

File: git/juansebastianform.html (hunk 1 above)
Comment (juancho11gm) [reply git/juansebastianform.html]: Done. Can you review it?, please

**Pull request merged**